app/views/components/_contents-list.html.erb

```diff
@@ -16,20 +16,37 @@
       <%= t("content_item.contents") %>
     </h2>
     <ol class="app-c-contents-list__list">
-      <% contents.each do |contents_item| %>
+      <% contents.each_with_index do |contents_item, position| %>
         <li
           class="app-c-contents-list__list-item app-c-contents-list__list-item--<%= parent_list_item_modifier %>"
         >
-          <%= link_to contents_item[:href], class: 'app-c-contents-list__link' do %>
-            <%= format_numbers ? wrap_numbers_with_spans(contents_item[:text]) : contents_item[:text] %>
-          <% end %>
+          <% link_text = format_numbers ? wrap_numbers_with_spans(contents_item[:text]) : contents_item[:text] %>
+          <%= link_to_if contents_item[:href], link_text, contents_item[:href],
+               class: 'app-c-contents-list__link',
+               data: {
+                 track_category: 'contentsClicked',
```

Comment:
Does this match the events used to track contents lists back in Q1 2017? It'd be good to keep the analytics data consistent.

Reply:
I've spoken to Tim who supplied the analytics tags and he says this is based on that, but deliberately not identical and he's happy to proceed with this tagging, so I'll leave this as is.